### PR TITLE
LR import: Code simplification + Fix for rotation angle

### PR DESCRIPTION
**Pull request by leclercj**

### CODE REVIEW

**Comment by TurboGit** [comment]
Looks quite simpler indeed ! Will land in 3.0.1 now. Thanks.

**Comment by leclercj** [comment]
New commit added to fix https://github.com/darktable-org/darktable/issues/3237

File: src/develop/lightroom.c [hunk 1]
```diff
@@ -1020,6 +1021,48 @@ static void _handle_xpath(dt_develop_t *dev, xmlDoc *doc, int imgid, xmlXPathCon
     }
 }
 
+static inline float dabs(double a)
```
Comment (TurboGit) [inline src/develop/lightroom.c]: This is standard fabs().

Comment (leclercj) [reply src/develop/lightroom.c]: You're right. I'll modify that

**Comment by TurboGit** [comment]
Thanks for fixing that for next release! I'm not sure how it gets broken over the time but this is really welcome! Will merge now.

**Pull request merged**